### PR TITLE
Exclude babel-plugin-css-modules-transform from defaults

### PR DESCRIPTION
**Pull request by js62789**
We need to be able to build our react-components library without transpiling CSS import statements. Disabling a plugin in a preset is not supposed to be done, so we need to exclude the plugin by default. That means that any project rendering React with CSS Modules will need to explicitly enable CSS Modules transpilation by setting `BABEL_ENV=reactServer`. That means that this is a breaking change and requires a major version bump.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ module.exports = function (api, options) {
   api.assertVersion(7);
 
   const isWebpack = (options && options.webpack);
+  const isReactServer = (options && options.reactServer);
```
Comment (djMax) [inline index.js]: You verified this comes through properly with BABEL_ENV=reactServer?

Comment (js62789) [reply index.js]: I was just doing that and it turns out that it does not. I need to update this to check `api.env('reactServer')`. Does this mean that the `@gasbuddy/react` package is not compiling the "webpack version" properly?

**Comment by js62789** [comment]
Ok this has been implemented in https://github.com/gas-buddy/react-components/pull/315. Both builds and tests pass.

File: index.js (hunk 1 above)
Comment (djMax) [reply index.js]: It's possible that didn't survive the babel update, though that would be surprising.

File: index.js [hunk 2]
```diff
@@ -13,7 +13,8 @@
 module.exports = function (api, options) {
   api.assertVersion(7);
 
-  const isWebpack = (options && options.webpack);
+  const isWebpack = (options && options.webpack) || api.env('webpack');
+  const isReactServer = (options && options.reactServer) || api.env('reactServer');
```
Comment (djMax) [inline index.js:17]: Any reason to keep both methods of lookup? This is a major version bump, so if it's cleaner to look for api.env, just do that.

**Review by djMax — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (js62789) [reply index.js:17]: Webpack uses options, but `@gasbuddy/react` uses `BABEL_ENV`, so it was for supporting both styles.

**Pull request merged**